scripts/cli/lec.sh

```diff
@@ -256,11 +256,51 @@ LIFERAY_WORKSPACE_HOME="$HOME/.liferay/workspace"
 
 RELEASES_JSON_FILE="${LIFERAY_WORKSPACE_HOME}/releases.json"
 
+RELEASES_JSON_URLS=(
+	"https://releases-cdn.liferay.com/releases.json"
+	"https://releases.liferay.com/releases.json"
+)
+
+_verifyReleasesJsonFile() {
+	if jq . "${RELEASES_JSON_FILE}" &>/dev/null; then
```

Comment:
should we keep track of the things that devs will need on their machines for `lec` to work correctly? i can't remember if this is the first time that we've introduced `jq`, but this is the first that i've seen of `jq`.